### PR TITLE
DX-498 updates to home page, audit fixes, and images

### PR DESCRIPTION
**Pull request by dtolb**
## DX-537
* http://localhost:4000/voice/methods/about.html
  * removed messaging from header
* http://localhost:4000/messaging/ratelimits.html#default-rate-limit
  * Added queue math image
* http://localhost:4000/messaging/callbacks/messageEvents.html
  * added clarity on events as list/array
* http://localhost:4000/messaging/methods/messages/createMessage.html
  * Remove old url
  * Updated FAQ to support.bandwith
* http://localhost:4000/messaging/methods/about.html
  * added media links

## DX-498
* Tweaked home page code examples to only curl as we wait on SDKs
* Added placeholders for SDK
* Changed some messaging language
* Change `userId` to `accountId` where applicable

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-home-page.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-home-page.s3-website-us-east-1.amazonaws.com/

File: layouts/header.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
        </div>
      </div>
    <a href="https://www.bandwidth.com" class="aimg"><div class="headerLogo"></div></a>
-   <h2><a href="http://bandwidth-docs.s3-website-us-east-1.amazonaws.com/">Docs for Developers</a> &nbsp;&#8250;&nbsp; Bandwidth APIs</h2>
+   <h2><a href="https://new.dev.bandwidth.com">Docs for Developers</a> &nbsp;&#8250;&nbsp; Bandwidth APIs</h2>
```
Comment (jmulford-bandwidth) [inline layouts/header.html]: Does this cover https://bandwidth.codefactori.com/browse/DX-555? Just want to track what else is completed by this change

Comment (dtolb) [reply layouts/header.html]: yes

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-home-page.s3-website-us-east-1.amazonaws.com/

File: messaging/methods/messages/createMessage.md [hunk 2]
```diff
@@ -8,26 +8,26 @@ Endpoint for sending text messages and picture messages using V2 messaging.
 
 <code class="post">POST</code>`https://messaging.bandwidth.com/api/v2/users/{userId}/messages`
 
-** Notice: The old URL `https://api.catapult.inetwork.com/v2/users/{userId}/messages` has been deprecated, and will be taken offline June 13th, 2019. Please update your system to use the new URL. **
-
 ### Invalid Phone Number handing
 
-When sending a group message to an invalid phone number, you may receive extraneous [callback events](../../callbacks//messageEvents.md).  You can read more on the [Messaging API concepts page](../concepts.md#group-message-invalid).
+When sending a group message to an invalid phone number, you may receive extraneous [callback events](../../callbacks/messageEvents.md).  You can read more on the [Messaging API concepts page](../concepts.md#group-message-invalid).
```
Comment (jmulford-bandwidth) [inline messaging/methods/messages/createMessage.md]: I think I missed this one on the audit I did. Can you change the concepts link to `../../about.md#group-message-invalid`? That one is correct

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: layouts/header.html (hunk 1 above)
Comment (jmulford-bandwidth) [reply layouts/header.html]: Cool. Let's make sure to close it out when this PR is merged

File: messaging/methods/messages/createMessage.md (hunk 2 above)
Comment (dtolb) [reply messaging/methods/messages/createMessage.md]: done

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-update-home-page.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — APPROVED** [review]

**Pull request merged**